bentoml/deployment/serverless.py

```diff
@@ -92,3 +99,31 @@ def deploy_with_serverless(platform, archive_path, extra_args):
     output_path = generate_serverless_bundle(bento_service, platform, archive_path, extra_args)
     subprocess.call(['serverless', 'deploy'], cwd=output_path)
     return output_path
+
+
+def delete_serverless_deployment(platform, archive, region, stage):
+    bento_server = load(archive)
```

Comment:
nit: * bento_service

Reply:
Actually, it looks like you don't need to load the entire archve, just need to load the config:
```
service_name =  load_bentoml_config(archive_path)["metadata"]["service_name"]
```

Reply:
done